### PR TITLE
Task/14817 adapt changes to certificates

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
Ramp Down Changes for the Certificates.
- Proof of status tile hidden
- Proof of status tile hidden
- "Check Validity for Travel" button hidden
- Re-issuing of certificates shall be hidden

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14817

## Screenshots
https://user-images.githubusercontent.com/72390277/220337394-a5535180-9ae3-446d-a02b-f9359d463869.MP4

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewModel.swift [hunk 1]
```diff
@@ -120,6 +120,7 @@ final class FooterViewModel {
 		switch button {
 		case .primary:
 			isPrimaryButtonEnabled = isEnabled
+			isPrimaryButtonHidden = CWAHibernationProvider.shared.isHibernationState
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewModel.swift]: Missing if check here

**Review by naveeddotio — CHANGES_REQUESTED** [review]
See comment

File: src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewModel.swift (hunk 1 above)
Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/FooterViewController/FooterViewModel.swift]: Added

**Review by naveeddotio — APPROVED** [review]

**Review by naveeddotio — CHANGES_REQUESTED** [review]
Tests are missing

**Review by flxschmidt — APPROVED** [review]

**Review by 30mar — APPROVED** [review]

**Pull request merged**